<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogViewForm</class>
 <widget class="QDialog" name="LogViewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro de Eventos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icons/icons/filter.png</normaloff>:/icons/icons/filter.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Desde</string>
     </property>
     <property name="indent">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hasta</string>
     </property>
     <property name="indent">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tipo de Evento</string>
     </property>
     <property name="indent">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Usuario</string>
     </property>
     <property name="indent">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="2" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Por ID</string>
     </property>
     <property name="indent">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="0" column="6" rowspan="2">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="indent">
      <number>4</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QDateTimeEdit" name="startDateTimeEdit">
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="displayFormat">
      <string>d 'de' MMM 'de' yyyy h:mm AP</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QDateTimeEdit" name="endDateTimeEdit">
     <property name="minimumSize">
      <size>
       <width>210</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="displayFormat">
      <string>d 'de' MMM 'de' yyyy h:mm AP</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="eventTypeComboBox">
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="userComboBox">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QComboBox" name="byIdComboBox">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLineEdit" name="idLineEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <widget class="QTableView" name="logTableView"/>
   </item>
   <item row="4" column="0" colspan="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>864</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="5" colspan="2">
    <widget class="QPushButton" name="filterButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Filtrar</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/icons/filter.png</normaloff>:/icons/icons/filter.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
